### PR TITLE
accessibility changes

### PR DESCRIPTION
**Pull request by khangmach**
Accessibility changes. Adding image alts. Deleting duplicate IDs. New nalanda bkg image. 

@jtamiace can plz review

### CODE REVIEW

File: fle_site/apps/main/templates/main/partials/_kickstarter_quote.html [hunk 1]
```diff
@@ -10,9 +10,9 @@
     <br>
     <div class="quote_2">
         <img id="quote_mark_l" src="{% static 'img/kickstarter/quote/quote.png' %}">
-        <p id="the_quote_2">I want to become an eye specialist...</p><br>
-        <p id="the_quote_2">because some people can't see how beautiful</p><br>
-        <p id="the_quote_2">is our world. So I want to show them.</p>
```
Comment (jtamiace) [inline fle_site/apps/main/templates/main/partials/_kickstarter_quote.html]: previously 3 id's on one page.......... sigh

Comment (khangmach) [inline fle_site/apps/main/templates/main/partials/_kickstarter_quote.html]: :)

**Comment by jtamiace** [comment]
Looks good to me. Like the new color for navbar. Pinging Radina for any other comments before I merge

**Comment by khangmach** [comment]
There are a few more changes to be made when time permits. The main change here that needs to be made asap is the nalanda background image change here. From 3-4mb to ~300kb. Dont wanna eat up too much data for mobile users

**Comment by jtamiace** [comment]
Ah, nice. Yeah that is huge. Radina has given ok to merge (cannot tag her)

**Pull request merged**